app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -164,6 +164,7 @@ class BrowserFragment : IWebViewLifecycleFragment(),
     override fun isBackEnabled() = canGoBack()
     override fun isForwardEnabled() = canGoForward()
     override fun getCurrentUrl() = url
+    override fun isURLPinned() = if (url != null) CustomTilesAccessor.isURLPinned(url!!) else false
```

Comment:
nit: url can be mutated while this method is called so it'd be better to do (for a one-liner):
```kotlin
url?.let { CustomTilesAccessor.isUrlPinned(it) } ?: false
```

But maybe it's best not written as one line